python/taichi/lang/quant_impl.py

```diff
@@ -5,6 +5,10 @@
 
 
 class Quant:
+    """Generator of quantized types.
```

Comment:
Could you also document the methods within this class?

Reply:
done